problem_builder/public/css/dashboard.css

```diff
@@ -24,7 +32,7 @@
     min-width: 4em;
     text-align: right;
     padding-right: 5px;
-    border-right: 0.6em solid transparent;
+    border-right: 1.875em solid transparent;
```

Comment:
@Kelketek cell height is 21.6 + 2*3.2 (padding) + 2*1 (border) = 30. Font-size or line-height, or whatever is equal to 1em for the cell is 16px; 30/16 = 1.875;

I've started with 2em and doesn't look more square to me :)